practice-one/assets/css/main.css

```diff
@@ -62,38 +62,38 @@
 }
 .login-button {
   display: flex;
-  padding: 16px 32px;
   align-items: flex-start;
+  padding: 16px 32px;
   gap: 8px;
-  border: none;
-  border-radius: 100px;
   background: var(--white, #fff);
   color: var(--Dark, #222831);
   font-family: Circular Std Book;
   font-size: 14px;
   font-style: normal;
   font-weight: 700;
   line-height: 120%;
+  border-radius: 100px;
```

Comment:
@quanghuy1799 Please check carefully the order of the organized CSS. I think the order of border property should be before the typography (font-*).